spec/actions/query_spec.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Metrics/BlockLength
 # frozen_string_literal: true
 
 RSpec.describe 'actions/query', :vcr do # rubocop:disable Metrics/BlockLength
```

Comment:
same here

Reply:
The `# rubocop:disable Metrics/BlockLength` has been removed